.editorconfig

```diff
@@ -20,3 +22,9 @@ insert_final_newline = false
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.go]
+indent_size = 8
```

Comment:
Ident size 8 imho is way too much 😰

Reply:
That section can just be removed, we use size 2 for go since a while now inherited from `[*]`.